packages/editor/package.json

```diff
@@ -79,6 +79,7 @@
     "react-colorful": "^5.6.1",
     "redent": "^4.0.0",
     "refractor": "^4.8.1",
+    "simplebar-react": "^3.3.2",
```

Comment:
[simplebar](https://github.com/Grsmto/simplebar/tree/master) is working great out of the box